packit_service/worker/handler.py

```diff
@@ -114,7 +114,7 @@ def report(
     def set_status(self, state: str, description: str, check_name: str):
```

Comment:
there is a set_commit_status call above - can the code above actually call this method so that we deduplicate?